docs/crd_mysql_backup_schedule.md

```diff
@@ -0,0 +1,44 @@
+
+`BackupSchedule` creates `CronJob` which collects binlog and then rotate dump.
+
+| Field        | Type                                          | Description                                          |
+| ------------ | --------------------------------------------- | ---------------------------------------------------- |
+| `apiVersion` | string                                        | APIVersion.                                          |
+| `kind`       | string                                        | Kind.                                                |
+| `metadata`   | [ObjectMeta]                                  | Standard object's metadata.                          |
+| `spec`       | [BackupScheduleSpec](#BackupScheduleSpec)     | Specification of scheduling.                         |
+| `status`     | [BackupScheduleStatus](#BackupScheduleStatus) | Most recently observed status of the scheduled jobs. |
+
+## BackupScheduleSpec
+
+| Field                    | Type                                    | Required | Description                                                               |
+| ------------------------ | --------------------------------------- | -------- | ------------------------------------------------------------------------- |
+| `clusterName`            | string                                  | Yes      | Name of [`MySQLCluster`](crd_mysql_cluster.md)                            |
+| `schedule`               | string                                  | Yes      | Schedule in Cron format, this value is passed to `CronJob.spec.schedule`. |
+| `objectStorageEndpoint`  | [ObjectStorageSpec](#ObjectStorageSpec) | Yes      | Specification of S3 compatible object storage.                            |
+| `retentionPeriodSeconds` | int                                     | Yes      | Retention period of each backup file.                                     |
+
+## BackupScheduleStatus
+
+| Field                | Type                                                    | Description                                    |
+| -------------------- | ------------------------------------------------------- | ---------------------------------------------- |
+| `lastCompletionTime` | [Time]                                                  | Completion time of the last backup.            |
+| `succeeded`          | boolean                                                 | `True` when the job is completed successfully. |
+| `conditions`         | [][`BackupScheduleCondition`](#BackupScheduleCondition) | The array of conditions.                       |
+
+## BackupScheduleCondition
+
+| Field                | Type   | Required | Description                                                      |
+| -------------------- | ------ | -------- | ---------------------------------------------------------------- |
+| `type`               | string | Yes      | The type of condition.                                           |
```

Comment:
should be an enum.

Reply:
Changed the type to Enum.
Possible values are not defined for now, so left as TBD.